### PR TITLE
Mes 7199 Make 1 Normal Stop the mandatory requirement for Cat D tests

### PR DESCRIPTION
**Pull request by solomonsimukoko**
Make 1 Normal Stop the mandatory requirement for Cat D tests

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2021-09-13 at 14 50 35

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -228,4 +228,4 @@
       "npm run lint:fix"
     ]
   }
-}
+}
```
Comment (Cullima) [inline package.json]: can you revert this change please

File: capacitor.config.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (Cullima) [inline capacitor.config.json]: can you revert this change please

File: src/providers/test-report-validator/__mocks__/test-result.mock.ts [hunk 3]
```diff
@@ -270,6 +270,7 @@ export const validTestCatC1E: CatC1EUniqueTypes.TestData = {
 
 export const validDelegatedTestCatDAndD1: CatD1UniqueTypes.TestData | CatDUniqueTypes.TestData = {
   testRequirements: {
+    normalStart1: false,
```
Comment (Cullima) [inline src/providers/test-report-validator/__mocks__/test-result.mock.ts:273]: should be able to leave delgated as is?

File: capacitor.config.json (hunk 2 above)
Comment (solomonsimukoko) [reply capacitor.config.json]: done

File: package.json (hunk 1 above)
Comment (solomonsimukoko) [reply package.json]: done

**Review by andrewsetterfield — APPROVED** [review]

**Review by Cullima — APPROVED** [review]
